resources/views/layout/app.blade.php

```diff
@@ -76,22 +79,50 @@ class="fal fa-folder fa-fw"></i> Folder</a>
     </div>
 </nav>
 <div class="container mt-5">
-    @if(Request::get('error')==1)<div class="alert alert-danger alert-dismissible fade" role="alert">
-        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-            <span aria-hidden="true">&times;</span>
-        </button>
-        <strong>You are not allowed to access to this resource</strong>
-    </div>@endif
+    <div id="divAlert"></div>
     @yield('content')
+
 </div>
 @yield('script')
 <script>
-    $(document).ready(function(){
-        $(".alert").addClass('show');
-    });
-    setTimeout(function(){
-        $(".alert").alert("close");
-    },5000);
+    function createAlert(alertType, alertText, linkHref = "", linkText = "") {
+        $(".alert").alert('close');
+        if (typeof alertTimeout !== 'undefined') {
+            clearTimeout(alertTimeout);
+        }
+        let divAlert = document.getElementById("divAlert");
+        let alert = document.createElement("div");
+        alert.classList.add("alert", alertType, "alert-dismissible", "fade");
+        let btn = document.createElement("button");
+        btn.classList.add("close");
+        btn.setAttribute("data-dismiss", "alert");
+        btn.setAttribute("aria-label", "Close");
+        let cross = document.createElement("span");
+        cross.setAttribute("aria-hidden", "true");
+        cross.appendChild(document.createTextNode("x"));
+        btn.appendChild(cross);
+        alert.appendChild(document.createTextNode(alertText));
+        let link = document.createElement("a");
+        link.setAttribute("href", linkHref);
+        link.appendChild(document.createTextNode(linkText));
+        link.classList.add("alert-link");
+        link.setAttribute("onClick", "$('.alert').alert('close');");
```

Comment:
Since you like playing with the DOM, go the extra mile.

```js
link.addEventListener("click", function(event){ ... });
```

Reply:
Thank's changed in last commit